### PR TITLE
DOC-3808 Update key concept page Ledger

### PR DESCRIPTION
**Pull request by michelemarsden**

### CODE REVIEW

File: content/en/platform/corda/4.8/open-source/key-concepts-ledger.md [hunk 1]
```diff
@@ -20,52 +20,50 @@ title: Ledger
...
-You can think of this vault as being a database or simple table.
+You can think of a vault as being a database or simple table. In this diagram, facts 1 and 7 appear on both Alice's
+vault and Bob's vault, and are therefore shared facts:
 
 {{< figure alt="ledger table" width=80% zoom="/en/images/ledger-table.png" >}}
-Where the rows are shared between nodes (facts 1 and 7 in this example), these are shared facts.
 
-Corda guarantees that whenever one of these facts is shared by multiple nodes on the network, it evolves in lockstep in the database of every node that is aware of it. This means that Alice and Bob will both see an **exactly identical version** of shared facts 1 and 7.
+Corda guarantees that when a fact is shared across multiple nodes on the network, it evolves in lockstep in each of
+these nodes' vaults. This means that Alice and Bob will both see an *exactly identical version* of shared facts 1 and 7.
```
Comment (erinking-r3) [inline content/en/platform/corda/4.8/open-source/key-concepts-ledger.md:61]:
```suggestion
When multiple nodes on a network share an evolving fact, the changes to the fact update at the same time in each node's vault. This means that Alice and Bob will both see an *exactly identical version* of shared facts 1 and 7.
```

File: content/en/platform/corda/4.8/open-source/key-concepts-ledger.md [hunk 2]
```diff
@@ -20,52 +20,50 @@ title: Ledger
...
 
-**Note:** Not all on-ledger facts are shared between peers. For example, Alice's fact 11 is not shared with Bob. Fact 11 could, in fact, not be shared with any other node at all. If this is the case, it is deemed a unilateral fact.
+On-ledger facts don't have to be shared between nodes. For example, fact 11 in Alice's vault is not shared with Bob.
+Facts that are not shared are *unilateral facts*.
 
-**Note:** Although there is no central ledger, it is possible to broadcast a basic fact to all participants should you wish to. You would do this by using the network map service to loop over all parties.
+{{< note >}}
+Although there is no central ledger, you can broadcast a basic fact to all nodes of a network using the network map
```
Comment (erinking-r3) [inline content/en/platform/corda/4.8/open-source/key-concepts-ledger.md:67]: I might give an example/use case for this to further clarify what that means, and also link to the network map page.

File: content/en/platform/corda/4.8/open-source/key-concepts-ledger.md [hunk 3]
```diff
@@ -20,52 +20,50 @@ title: Ledger
 
 ## Summary
```
Comment (erinking-r3) [inline content/en/platform/corda/4.8/open-source/key-concepts-ledger.md:21]: I think what's missing from this topic is a very general introduction to the concept of a ledger (just a simple one-line intro, such as "A ledger represents the current state of affairs as a journal of transactions.") and a short introduction to the concept of decentralized ledgers. There are some good things in this article from Marco Polo that you could include, that are specific to Corda https://marcopolonetwork.com/articles/distributed-ledger-technology/

I might also include a short note paying lip service to DLT vs blockchain

**Review by erinking-r3 — COMMENTED** [review]
Vast improvement! Left a few comments.

File: content/en/platform/corda/4.8/open-source/key-concepts-ledger.md [hunk 4]
```diff
@@ -20,52 +20,50 @@ title: Ledger
...
 
-* *The ledger is subjective from each peer’s perspective*
-* *Two peers are always guaranteed to see the exact same version of any on-ledger facts they share*
+* *The ledger refers to all facts that are shared between the nodes of a network.*
```
[the remark this answers is not in the document]
Comment (michelemarsden) [reply content/en/platform/corda/4.8/open-source/key-concepts-ledger.md:23]: @erinking-r3, I thought on the Networks page you revised, you kept the italics?

File: content/en/platform/corda/4.8/open-source/key-concepts-ledger.md [hunk 5]
```diff
@@ -20,52 +20,48 @@ title: Ledger
 
 ## Summary
 
-* *The ledger is subjective from each peer’s perspective*
-* *Two peers are always guaranteed to see the exact same version of any on-ledger facts they share*
+* A distributed ledger is a database of facts that's replicated, shared, and synchronized across multiple participants.
```
Comment (michelemarsden) [inline content/en/platform/corda/4.8/open-source/key-concepts-ledger.md:23]:
```suggestion
* A distributed ledger is a database of facts that's replicated, shared, and synchronized across multiple participants on a network.
```

**Review by erinking-r3 — APPROVED** [review]
LGTM!

**Pull request merged**